### DISCUSSION TITLE
How to style text?

### DISCUSSION THREAD

**Discussion by vnugent** [saadeghi/daisyui#1040] [Q&A] [answered]
Our project is a  [wiki](https://openbeta.io) which has a lot of text content (we're using Tailwind but without the typography plugin).  

Before adopting Daisy we use custom classes `text-primary`, `text-secondary` `text-tertiary` for text, which are neutral colors in Tailwind, text-gray-900, text-gray-600, text-gray-400 respectively.  Those custom classes won't work in Daisy because primary/secondary are now brand colors and we still want gray/neutral color for text.

What would you recommend a semantic way to convey primary and secondary styles for text?

**Comment by saadeghi** [comment, marked as answer]
Check this page to see all color names:  
https://daisyui.com/docs/colors/

You can still use `text-primary` and `text-secondary` and they will use primary and secondary colors of the current daisyUI theme.  
You might also need `text-base-content` which is the default text color.

And you can still use any Tailwind CSS color or add your own colors. There are no limits. daisyUI just provides a set of additional colors.

Let me know if you have any questions.

**Comment by vnugent** [comment]
Thanks.  It's not immediately obvious, but now I see `text-base-content` `text-base-300` , `text-base-200`, `text-base-100` would serve my needs.
